pkg/crc/ssh/client.go

```diff
@@ -24,7 +24,7 @@ type NativeClient struct {
 	Port     int
 	Keys     []string
 
-	conn *ssh.Client
+	sshClient *ssh.Client
```

Comment:
not sure which name is best to be honest, having something named `sshClient` in a type defined in `ssh/client.go` is imo not great.
Edit: I see there's another member named `conn` in the next commit, always useful to explain why you are doing something in the log to save some review time, and avoid this kind of comments ;)